augur/cli/user.py

```diff
@@ -39,25 +39,36 @@ def __init__(self, **kwargs):
 
     def __repr__(self):
         return (
-            f"<User username={self.username}, public_id={self.public_id}, admin={self.admin}>"
+            f"<User username={self.username}, email={self.email}, public_id={self.public_id}, admin={self.admin}>"
         )
 
-@click.group('user', short_help='Add user utilities')
+#create user table
+db.create_all()
+
+#user subcommand
+@click.group('user', short_help='Support for adding regular users or administrative users')
 def cli():
     pass
 @cli.command('add', short_help="Add a new user")
 @click.argument("username")
+@click.argument("email")
 @click.option(
     "--admin", is_flag=True, default=False, help="New user has administrator role"
```

Comment:
to work on: 
use a different model instead of flag for admin, will be easy to differentiate between user and admin